common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java

```diff
@@ -715,7 +719,12 @@ public void addTopicPartitions(String topic, int partitions) {
             throw new IllegalArgumentException("consumerGroup cannot be null, empty or blank");
 
         try {
-            return getAdminClient().describeConsumerGroup(consumerGroup);
+            AdminClient.ConsumerGroupSummary summary = getAdminClient().describeConsumerGroup(consumerGroup, operationTimeout);
+            if (summary.state().equals(ConsumerGroupState.DEAD.toString())) {
+                throw new AdminOperationException("Consumer group not found: " + consumerGroup);
```

Comment:
Add doc about this case

Reply:
It's been a couple weeks but I think I added this to preserve the original behavior... agree it is probably better to return the DEAD group summary and just fix the tests and whatever code is calling this to check for dead

Reply:
Done (both) - https://github.com/cerner/common-kafka/commit/8d357d9b12a0fca17910b0bb3728fb89030cfd2e